n_particles: 100
z_hit: 0.5